### ISSUE TITLE
Change sliders on home page

### ISSUE THREAD

**Issue by hwasserstrum** [chrisflesher/rcvfortexas#78]
I tried to create a new slider and started a new page called try_RCV_pres. I am stuck. I want to direct viewers to this url: [github.com/rcvfortexas/rcvfortexas](https://app.rankedvote.co/rv/4ib3bswrj4s5g0e87o/vote) and use this image: 
![Rank_the_Candidates!](https://github.com/rcvfortexas/rcvfortexas/assets/88623190/71529e16-f113-46d2-82a5-b031b6dae704)
For the tagline: The list of Republican presidential hopefuls is growing. Express your preferences using RCV. 

I'd like two more new sliders. Since I hadn't figured out how to do it, I didn't start new pages for these. 
1- One for the poll https://app.rankedvote.co/rv/df145v2x0ww8v113hi/vote. Here is an image we can use: 
![Why voters love RCV](https://github.com/rcvfortexas/rcvfortexas/assets/88623190/706ff20f-e1c1-4abe-99f9-98e8d97ae4ac)
For the tagline: Help us count the ways voters support RCV.
2- One for a more recent Action Alert. Here is the URL to direct people to: https://p2a.co/kT7Lmx1. For the image: 
![capitol building 6](https://github.com/rcvfortexas/rcvfortexas/assets/88623190/b4c207b1-11ff-4ec2-8668-72abded5d5f8)

For the tagline: Take this 3-question survey to let your elected officials you want RCV.

Finally, please remove the Stop the Ban and Action Alert sliders now on the website.
Thanks!

**Comment by dsgit215** [comment]
The 3 new messages have been added to the "Featured content sliders" on the Home page.  Also, the previous two messages have been archived.  

For more information about the "Featured content sliders" on the home page, see - [https://support.nationbuilder.com/en/articles/2319387-configure-featured-content-sliders]

NOTE: I've put all imagery and text content within the slider image as opposed to using the 'Headline / Action text' field in the "Featured content sliders" form.  While I did this to have greater control over presentation, note that this means the text content is not visible to search engines.  Also, since the link for each message goes to an external site, it was necessary to create "redirect" pages so internal slugs could be used in the "Page to feature" field of the slider form.
